firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandlerTest.java

```diff
@@ -766,6 +768,20 @@ public void getInfo_hitsThrottleLimit_throttledStatus() throws Exception {
         .isEqualTo(firstFetchedContainer.getFetchTime());
   }
 
+  @Test
+  public void customSignals_updated_onSubsequentFetch() throws Exception {
+    Map<String, String> customSignals =
+        ImmutableMap.of(
+            "subscription", "premium",
+            "age", "20");
+    sharedPrefsClient.setCustomSignals(customSignals);
+    fetchCallToHttpClientUpdatesClockAndReturnsConfig(firstFetchedContainer);
+
+    assertWithMessage("Fetch() failed!").that(fetchHandler.fetch().isSuccessful()).isTrue();
```

Comment:
This assertion seems unrelated to the test case, can we remove it?

Reply:
I have removed the assertion and am now calling fetch directly.